_pages/about-us/teams/products-and-platforms.md

```diff
@@ -1,10 +1,12 @@
 ---
 title: Products and Platforms
 ---
-Products and Platforms is one of the 5 business units within 18F. We strive to incubate tools for digital government.
```

Comment:
I'm not sure about the use of _incubate_, which is jargony and could alienate some readers. What alternatives would you all consider instead? (Perhaps build, generate the concepts for, test ...?)